### DISCUSSION TITLE
How can I make the top bar transparent?

### DISCUSSION THREAD

**Discussion by IcyAlmond** [NvChad/base46#356] [Q&A] [answered]
<img width="1028" alt="{60084553-BAA3-45D6-BECD-5541AC934758}" src="https://github.com/user-attachments/assets/0425ca44-9fc4-4a2c-9ef9-90f547eb259d">
Running `:hi TabLine guibg=NONE` gets the whole top bar to go transparent, but I don't know how to set this in the configuration files.

This is my chadrc.lua which shows what i've tried so far. 
```lua
M.base46 = {
  theme = "rxyhn",
  transparency = true,
  changed_themes = {
    rxyhn = {
      base_30 = {
        black = "#000000", -- usually your theme bg
        grey = "#14d17c",
        grey_fg = "#AAAA88",
        grey_fg2 = "#dad7ee",
        line = "#7F7F7F",
      },
      tbline = {
        TbFill = { fg = "NONE", bg = "NONE" },
        TbBufOn = { fg = "NONE", bg = "NONE" },
        TbBufOff = { fg = "NONE", bg = "NONE" },
        TbBufOnModified = { fg = "NONE", bg = "NONE" },
        TbBufOffModified = { fg = "NONE", bg = "NONE" },
        TbBufOnClose = { fg = "NONE", bg = "NONE" },
        TbBufOffClose = { fg = "NONE", bg = "NONE" },
        TbTabNewBtn = { fg = "NONE", bg = "NONE" },
        TbTabOn = { fg = "NONE", bg = "NONE" },
        TbTabOff = { fg = "NONE", bg = "NONE" },
        TbTabCloseBtn = { fg = "NONE", bg = "NONE" },
        TBTabTitle = { fg = "NONE", bg = "NONE" },
        TbThemeToggleBtn = { fg = "NONE", bg = "NONE" },
        TbCloseAllBufsBtn = { fg = "NONE", bg = "NONE" },
      }
    }
  },
  integrations = { "tbline" },
  hl_override = {
    Comment = { italic = true },
    ["@comment"] = { italic = true },
    ["@TabLine"] = { fg="NONE", bg="NONE" },
    TabLine = { fg="NONE", bg="NONE" },
    TbFill = { fg="NONE", bg="NONE" },
    TbBufOn = { fg="NONE", bg="NONE" },
    TbBufOff = { fg="NONE", bg="NONE" },
    TbBufOnClose = { fg="NONE", bg="NONE" },
    TbBufOffClose = { fg="NONE", bg="NONE" },
    TbBufOnModified = { fg="NONE", bg="NONE" },
    TbBufOffModified = { fg="NONE", bg="NONE" },
    -- [LineNr] = { fg = "#000000" }
  }
}
```

I got those Tb* names from https://github.com/NvChad/base46/blob/v2.5/lua/base46/integrations/tbline.lua 
How am I supposed to configure the top bar? I was looking to get the bar background to be clear, but the tabs themselves different color, kind of like the second image in this pull request: https://github.com/NvChad/base46/pull/217

**Comment by siduck** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/fbfc3ad4-741f-4696-b01e-487939247e3e)

it works here, remove all your tabline hlgroups, not needed

**Reply by IcyAlmond** [reply]
Hm, for some reason it works on one of my devices but not the other, I have my nvim config saved to a git repo and they are both on the exact same commit but one has the top bar transparent and the other one is colored.

I'm not sure how to debug this, any ideas?

**Reply by siduck** [reply]
both use same nvim version?

**Reply by IcyAlmond** [reply]
Ohhhh you're right, one was compiled from git a few days ago. the stable release works. Fixed on both now, thanks
